src/Interactivity/woo-directives.php

```diff
@@ -0,0 +1,123 @@
+<?php
+include_once __DIR__ . '/../../../gutenberg/lib/experimental/html/wp-html.php';
```

Comment:
Note that the files in `lib/experimental/html` were recently moved to a different location. You might want to carry over the changes from https://github.com/WordPress/block-hydration-experiments/pull/162 😅